low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py

```diff
@@ -491,6 +496,11 @@ def _create_internal_msg_dg(self, alert_type, disk_group_detail):
                 extended_info.update({key : value})
 
         generic_info.update(extended_info)
+        cvg_info = {
+            "cvg_name": resource_id if resource_id in self.cvg_info_dict else "NA",
+            "cvg_index": self.cvg_info_dict.get(resource_id, "NA")
```

Comment:
as discussed, use cvg_id only as the key

Reply:
Done